src/app/workflow-editor/graph-editor/graph-editor/workflow-graph-editor.component.ts

```diff
@@ -554,6 +563,10 @@ export class WorkflowGraphEditorComponent extends DirectiveBase implements OnCha
             if (!path) {
                 return;
             }
+
+            const addExtension = path.split("/").slice(-1)[0].split(".").length === 1;
+            path = addExtension ? path + ".svg" : path;
```

Comment:
Paths can contain legit dots that do not imply extension. Maybe it would be better to check if `path.endsWith(".svg")` and add it if necessary.